sdk/data/azcosmos/cosmos_client.go

```diff
@@ -70,7 +73,7 @@ func NewClient(endpoint string, cred azcore.TokenCredential, o *ClientOptions) (
 	if err != nil {
 		return nil, err
 	}
-
+	log.Write(azlog.EventResponse, "\n===== Client configuration:\n"+o.String()+"\n=====\n")
```

Comment:
Our diagnostics in other SDKs print this on all operations, should we do the same here? Maybe before https://github.com/Azure/azure-sdk-for-go/blob/6289abf0dc0c0c6b34ba96ba385a135f34ab39a6/sdk/data/azcosmos/cosmos_client.go#L474 ?

Reply:
does it make sense for us to though? can the client settings be changed by the user without restarting their client? I'd think these settings hold throughout the lifecycle of the user's application right?

I understand that parity is a big thing, but based on this morning's conversation I think we should be good to deviate from that no?

Reply:
Imagine someone going through the logs, finding an operation that went to the "wrong" region, then filing an Issue with that complaint and including only the log for the operation. It would not be possible to know why?